LiteCore/Query/QueryParser.cc

```diff
@@ -706,8 +713,9 @@ namespace litecore {
             if (expr && expr[0]->asString().caseEquivalent("AS"_sl)) {
                 // Handle 'AS':
                 require(expr.count() == 3, "'AS' must have two operands");
-                title = string(requiredString(expr[2], "'AS' alias"));
+                title = unescapeAlias(string(requiredString(expr[2], "'AS' alias")));
```

Comment:
I think it's better to fail if the alias contains invalid characters (quotes or backslashes), rather than just removing them.
Also, for consistency the same rules should also apply to other "AS" aliases, i.e. in joins.